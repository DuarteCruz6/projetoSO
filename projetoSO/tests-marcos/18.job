WRITE [(o5,98)(e6,40)(a9,96)(i3,6)(a1,78)]
WRITE [(w8,77)(s3,7)(e7,11)(x1,89)(z8,100)]
WAIT 3000
WRITE [(b9,28)(n6,45)(n5,80)]
WRITE [(j2,70)]
READ [a1]
WRITE [(d4,43)]
WAIT 2000
WRITE [(m9,9)(g0,56)]
READ [n5]
WRITE [(t1,53)]
WRITE [(c4,33)(d6,9)(j1,85)(h0,48)(z0,16)]
SHOW
WRITE [(i4,17)(y9,40)(w2,100)]
