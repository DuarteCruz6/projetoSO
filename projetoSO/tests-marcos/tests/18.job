WAIT 1000
WRITE [(z1,39)(g0,91)(w3,57)(l5,52)(y1,71)]
WRITE [(z2,66)(f2,80)(u8,84)(q3,99)(z2,17)]
READ [w3]
WRITE [(v6,90)(p2,100)(d0,38)(g4,18)(r0,62)]
DELETE [z2,f2,l5,g0,g4]
WRITE [(j8,45)(r9,74)]
DELETE [r9,r0,v6]
WAIT 2000
READ [q3]
READ [w3]
WRITE [(d1,60)(d9,25)]
DELETE [w3,j8,d9,z1]
WAIT 1000
