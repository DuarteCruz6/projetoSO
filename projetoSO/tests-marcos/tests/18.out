[(w3,57)]
[(q3,99)]
[(w3,57)]
